src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/dto/AnalysisSistrResults.java

```diff
@@ -0,0 +1,42 @@
+package ca.corefacility.bioinformatics.irida.ria.web.analysis.dto;
+
+import java.util.Map;
+
```

Comment:
File comment

Reply:
Added in 453b37f923a9910ec92e6dda3134f7bb7ff0d8d7

Reply:
Added in 453b37f923a9910ec92e6dda3134f7bb7ff0d8d7